drush.php

```diff
@@ -38,6 +40,61 @@
  *     - Run commands and command hooks via annotated commands library
  *     - Catch 'command not found' exception, bootstrap Drupal and run again
  *   - Return status code
+ *
+ * ## Viable Drush configurations ##
+ *
+ * As of Drush 12, only a site-local Drush will bootstrap Drupal.
+ * A globally installed Drush is no longer supported.
+ *
+ * The following directory layouts are supported:
+ *
+ * Drush binary in site-local configuration in recommended Drupal site: (typical)
+ *
+ *         drupal
+ *         ├── web
+ *         │   ├── core
+ *         │   └── index.php
+ *         └── vendor
+ *             ├── autoload.php
+ *   [*1]      ├── bin
+ *             │   └── drush.php -> ../drush/drush/drush.php
+ *   [*2]      └── drush
+ *                 └── drush
+ *                     └── drush.php
+ *
+ * Drush binary in site-local configuration in a legacy Drupal site: (unusual)
+ *
+ *         drupal
+ *         ├── core
+ *         ├── index.php
+ *         └── vendor
+ *             ├── autoload.php
+ *   [*1]      ├── bin
+ *             │   └── drush.php -> ../drush/drush/drush.php
+ *   [*2]      └── drush
+ *                 └── drush
+ *                     └── drush.php
+ *
+ * Drush project: (Only used when developing Drush; `composer install --dev`)
```

Comment:
Maybe remove `composer install --dev`? I'm not sure how it distinguishes from a regular site.